dist/index.js

```diff
@@ -477,6 +477,8 @@ function setupWindows(version, cmdExeWorkaround, githubAuth) {
         }
         const url = `download.clojure.org/install/win-install${version === 'latest' ? '' : `-${version}`}.ps1`;
         yield exec.exec(`powershell -c "iwr -useb ${url} | iex"`, [], {
```

Comment:
I think using `powershell -c` is redundant here.

```suggestion
        yield exec.exec(`iwr -useb ${url} | iex`, [], {
```